lib/src/core/number_to_words/number_to_words.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:persian_tools/persian_tools.dart';
+import 'package:persian_tools/src/constants/number_to_words/constants.dart';
+import 'package:persian_tools/src/internal_methods.dart';
+
+/// gets an [int] with 3 or less digits as input and converts it to persian
+String numToWord(int number) {
```

Comment:
Make this function private if it's not supposed to be used directly in the package.